internal/pkg/service/buffer/worker/service/conditions_test.go

```diff
@@ -1,346 +0,0 @@
-package service_test
-
-import (
-	"context"
-	"testing"
-	"time"
-
-	"github.com/benbjohnson/clock"
-	"github.com/c2h5oh/datasize"
-	"github.com/keboola/go-client/pkg/keboola"
-	"github.com/keboola/go-utils/pkg/wildcards"
-	"github.com/stretchr/testify/assert"
-	etcd "go.etcd.io/etcd/client/v3"
-
-	"github.com/keboola/keboola-as-code/internal/pkg/idgenerator"
-	bufferDependencies "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model/column"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/worker/service"
-	"github.com/keboola/keboola-as-code/internal/pkg/service/common/dependencies"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/utils/testproject"
-)
-
-func TestConditionsChecker(t *testing.T) {
-	t.Parallel()
```

Comment:
Conditions su po novom E2E otestovane tu:

Upload:
- https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/001_import_records_test.go#L7-L8
- https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/002_slice_upload_test.go#L21-L22


Import:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/test/buffer/worker/003_file_import_test.go#L26-L27

Reply:
Tam ^^^ sa to testuje nad `rows count`, ale z E2E hladiska to je jedno.

Unit testy pre vsetky druhy conditions mame tu:
https://github.com/keboola/keboola-as-code/blob/718568f24aaf898fb528a35b4aba9dfadd71741b/internal/pkg/service/buffer/store/model/conditions_test.go#L56